src/widgets/archivelistwidget.cpp

```diff
@@ -4,6 +4,7 @@
 #include "ui_archiveitemwidget.h"
 
 #include <QMessageBox>
+#include <QKeyEvent>
```

Comment:
This was related to the `QMouseEvent` removal from `textlabel.h`.  If I didn't remove that, the other files compiled fine (presumably because they implicitly included `QMouseEvent`, although I'm not certain how that would supply `QKeyEvent` since according to the Qt docs it's not a direct inherited class).  I therefore thought it was related to this PR, since a PR should never leave the tree in an un-compilable state.